[Observability] Several fixes to the observability doc (#36054)
ray-project/ray: doc/source/ray-observability/getting-started.rst

The Metrics view provides visualizations of the time series metrics emitted by Ray.

You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.

> There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.

Reviewer: ```suggestion
There is also a convenient button to open the Grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
```

**Driver logs**

If the Ray Job is submitted by the :ref:`Ray Job API <jobs-quickstart>`, the Job logs are available from the Dashboard. The log file follows the following format; ``job-driver-<job_submission_id>.log``.

.. note::

>   If the driver is executed directly on the head node of the Ray Cluster (without the job API) or run with :ref:`Ray client <ray-client-ref>`, the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.

Reviewer: ```suggestion
  If you execute the driver directly on the head node of the Ray Cluster (without the Job API) or run with :ref:`Ray client <ray-client-ref>`, the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.
```

**Task and Actor Logs (Worker logs)**

> Task and actor logs are accessible from the :ref:`task and actor table view <dash-workflow-state-apis>`. Click the "Log" button.

Reviewer: ```suggestion
Task and Actor logs are accessible from the :ref:`task and actor table view <dash-workflow-state-apis>`. Click the "Log" button.
```
